example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
```

Comment:
Should we bump it for the library as well?

Reply:
Yes.

Reply:
I mean, it's not mandatory because everything works. But it will be good be aligned with Flutter.

Reply:
Can you take care of that in this PR?